console/src/test/java/org/georchestra/console/integration/UsersIT.java

```diff
@@ -48,20 +61,35 @@
 
     private @Autowired LdapTemplate ldapTemplateSanityCheck;
 
+    private @Autowired GDPRAccountWorker gdprWorker;
+
     @Autowired
     private WebApplicationContext wac;
 
     private MockMvc mockMvc;
 
+    private Set<String> createdUsers;
+
     public static @BeforeClass void init() {
     }
 
     public @Before void before() {
+        this.createdUsers = new HashSet<>();
         // pre-flight sanity check
         assertNotNull(ldapTemplateSanityCheck.lookup("cn=admin"));
         this.mockMvc = MockMvcBuilders.webAppContextSetup(this.wac).build();
     }
 
+    public @After void cleanUp() {
+        for (String user : new HashSet<>(createdUsers)) {
+            try {
+                deleteUser(user);
+            } catch (Exception e) {
+                e.printStackTrace();
```

Comment:
I know we are in an IT, but why not using a logger here ? I'm still surprised when I see a printStacktrace() call.

Reply:
good call, gonna fix it